lib/git-interface/comment.ts

```diff
@@ -0,0 +1,73 @@
+import { User } from "./user"
+
+export class GitComment
+{
+    public id: number;
+    public comment_id: string;
+    public url: string;
+    public html_url: string;
+    public body: string;
+    public user: User;
+    public created_at: string;
+    public updated_at: string;
+
+    constructor(c_body: string, id=0, comment_id="", url="", html_url="", user=new User(),
+        created_at="", updated_at="")
+    {
+        this.id = id;
+        this.body = c_body;
+        this.user = user;
+        this.comment_id = comment_id;
+        this.url = url;
+        this.html_url = html_url;
+        this.created_at = created_at;
+        this.updated_at = updated_at;
+    }
+}
+
+export class GitPRComment extends GitComment
+{
+    public path: string; // path to file that necessitates the comment
+    public diff_position: number;
+
+    constructor(c_body: string, id=0, comment_id="", url="", html_url="", user=new User(), 
```

Comment:
if you want, `prettier.ts` is working and you can autoformat with `npm run prettier`